Services/Interfaces/web_interface/tests/__init__.py

```diff
@@ -92,7 +92,10 @@ async def check_page_no_login_redirect(url, session):
         if not (url.endswith("login") or url.endswith("logout") or url.endswith("/community")):
             assert "input type=submit value=Login" not in text
             assert not resp.real_url.name == "login"
-        assert resp.status == 200
+        if url.endswith("historical_portfolio_value"):
```

Comment:
What's the issue here?

Reply:
no exchange manager is live of this test setup, therefore we should not return 200 as there is no exchange to be found